xds/internal/client/client_rds_test.go

```diff
@@ -280,7 +280,14 @@ func (s) TestRDSGenerateRDSUpdateFromRouteConfiguration(t *testing.T) {
 							Route: &v3routepb.RouteAction{
 								ClusterSpecifier: &v3routepb.RouteAction_Cluster{Cluster: clusterName},
 							}}}}}}},
-			wantError: true,
+			wantUpdate: RouteConfigUpdate{
```

Comment:
Hmm ... does it make sense for our internal types to match the xDS proto in the sense that we also store whether the match is `CaseSensitive` instead of the other way around?

Reply:
I used case **in**sensitive in the field because the default value of bool is false, and it's a sane default.

If I make it case sensitive, I will need to make it a `*bool`, so when it's not set, `nil` means case sensitive.

Reply:
> I used case insensitive in the field because the default value of bool is false, and it's a sane default.

It looks to me with the way the proto is structured that they want the default to be case sensitive. This is comment on the field:
```
  // Indicates that prefix/path matching should be case sensitive. The default
  // is true.
```